Improve and extend feed sync chapters for community containers (#144)
greenbone/docs: src/common/container/performing-feed-sync.md

## Performing a Feed Synchronization

For the actual vulnerability scanning, {term}`Vulnerability Tests<VT>`,
security information like CVEs, port lists and scan configurations are required.
All this data is provided by the {term}`Greenbone Community Feed` and
must be download and loaded initially before starting a vulnerability scan.

A synchronization always consists of two parts:

1. Downloading the changes via {command}`rsync`

> 2. Loading the changes by a daemon into memory and a database

Reviewer: ```suggestion
2. Loading the changes into memory and a database by a daemon
```

Reviewer 2: I would do sth. like
`2. The changes get loaded into memory and a database by a daemon`
because it sounds really weird like it is currently?

Author: I am fine with `The changes get loaded into memory and a database by a daemon`

> 2. Loading the changes by a daemon into memory and a database

Author: ```suggestion
2. The changes get loaded into memory and a database by a daemon
```

> Both steps may take a while, from several minutes up to hours. Especially for the

Reviewer: ```suggestion
Both steps may take a while, from several minutes up to hours, especially for the
```
